tools/clang/unittests/HLSL/ExecutionTest.cpp

```diff
@@ -5952,25 +6079,96 @@ void ExecutionTest::RunRawBufferLdStTest(D3D_SHADER_MODEL shaderModel, RawBuffer
        WEX::Logging::Log::Result(WEX::Logging::TestResults::Skipped);
        return;
      }
+     sTy = (dataType == RawBufferLdStType::I16 ? "int16_t" : "half");
      break;
    case RawBufferLdStType::I32:
+     sTy = "int32_t";
+     break;
    case RawBufferLdStType::Float:
+     sTy = "float";
      break;
    default:
      DXASSERT_NOMSG("Invalid RawBufferLdStType");
    }
 
-   if (shaderOpName == nullptr) {
-     // TODO: finish up all variations
-     WEX::Logging::Log::Result(WEX::Logging::TestResults::Skipped);
-     return;
-   }
-
+   // format shader text
+   std::string shaderText(RawBufferTestComputeShaderTemplate);
+   ReplaceAll(shaderText, std::string("DATATYPE_SIZE"), std::to_string(sizeof(Ty)));
+   ReplaceAll(shaderText, std::string("DATATYPE"), sTy);
```

Comment:
Suggestion: Add a prefix $ to those to make it clearer that they are replacement patterns.